app/controllers/events_controller.rb

```diff
@@ -0,0 +1,74 @@
+class EventsController < ApplicationController
+  before_action :set_event, only: %i[show edit update destroy]
+
+  # GET /events or /events.json
+  def index
+    @events = Event.all
+  end
```

Comment:
It should list the past and upcoming events. E,g 
def index
  @past_events = Event.previous_events
  @upcoming_events = Event.upcoming_events
end